etna/collections/migrations/0005_resultspagerecordpage.py

```diff
@@ -28,7 +28,7 @@ class Migration(migrations.Migration):
                     "sort_order",
                     models.IntegerField(blank=True, editable=False, null=True),
                 ),
-                ("record_iaid", models.TextField()),
+                ("record_metadataId", models.TextField()),
```

Comment:
Changing existing migrations that have already run is a no-no. This change should be reverted and taken care of in a new migration